pkg/republisher/uuid_republisher.go

```diff
@@ -62,7 +85,7 @@ func (r *NotifyingUUIDRepublisher) Republish(uuid, tidPrefix string, republishSc
 			return nil, errs
 		}
 		log.Infof("uuid=%v was found to be an ImageSet having an imageModelUUID=%v", uuid, imageModelUUID)
-		msg, isFound, err := r.ucRepublisher.RepublishUUIDFromCollection(imageModelUUID, tid, r.collections["methode"])
+		msg, isFound, err := r.ucRepublisher.RepublishUUIDFromCollection(imageModelUUID, tid, r.collections["universal-content"])
```

Comment:
I haven't spotted this change until now. Why the collection is changing from `methode` to `universal-content` for image republishing?

Reply:
This is the only remaining point to be cleared out. Why is this change made, is it necessary?

Reply:
This is something that i've mentioned and it needs to be discussed or at least reviewed.

As far as I understand whenever a UUID is not found in any collection then we check whether the UUID belongs to a image, in case it does - returns other UUID, it tries to republish it in ‘universal-content’  used to be ‘Methode’ the default collection.

I’ve made this change from Methode to Universal-content due the Methode decommissioning.

By migration and decommissioning AFAIK means :

1- Methode has been migrated from a bare metal to cloud.
2- Removed (WIP) from our architecture.
3- From now on it will be used only for paper editions.

Besides I’m not sure that this approach of republishing is still a valid one, one of the reasons I wanted this PR to be reviewed extensively.

Reply:
I don't see how this is related with the `list republish order` that we're addressing here. If you see the need for that change, please raise a different ticket, so it can be addressed separately. If you feel that there is something that needs to be discussed first, raise it during the daily or arrange a meeting.
But all we do now is to hold up this PR for an unclear and unrelated change.

Reply:
i'd leave it as it was.